db/schema.rb

```diff
@@ -10,9 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_08_17_100339) do
+ActiveRecord::Schema.define(version: 2018_08_29_153755) do
 
-  create_table "list_items", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+  create_table "internal_change_notes", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "author"
```

Comment:
You might want to consider using the `users` table, which is automatically populated when users login (through the gds-sso gem), rather than just storing the author as a string (assuming this is the author of the note).